admin_guide/manage_scc.adoc

```diff
@@ -34,6 +34,13 @@ As an {product-title} cluster administrator, you can list and view details for
 SCCs, but cannot edit or delete the default SCCs.
 endif::openshift-dedicated[]
 
+[IMPORTANT]
+====
+Do not modify the default SCCs. Customizing the default SCCs can lead to issues
+when upgrading. Instead, xref:creating-new-security-context-constraints[create
```

Comment:
@bfallonf - this should have been conditionalized in the openshift-enterprise,openshift-origin ifdef above. This breaks the build in dedicated as it doesn't contain the xref you have linked to.